### PR TITLE
Add simple logging via NLog.

### PR DESCRIPTION
**Pull request by TheSeems**

### CODE REVIEW

File: AMTGBot/Bot.cs [hunk 1]
```diff
@@ -11,12 +14,16 @@ public class Bot
         public ITelegramBotClient BotClient { get; }
         public BotConfig BotConfig { get; }
         public ILatexRenderer LatexRenderer { get; }
+        public Logger Logger { get; private set; }
```
Comment (WhiteBlackGoose) [inline AMTGBot/Bot.cs]: I'm lost, are we into DI or not? What I'm saying if we need 100% of SOLID, we might want to make them all interfaces.

File: AMTGBot/Bot.cs [hunk 2]
```diff
@@ -11,12 +14,16 @@ public class Bot
...
 
         public Bot(ITelegramBotClient botClient, BotConfig botConfig, ILatexRenderer latexRenderer)
         {
             BotClient = botClient;
             BotConfig = botConfig;
             LatexRenderer = latexRenderer;
+
+            Logger = LogManager.GetCurrentClassLogger();
```
Comment (WhiteBlackGoose) [inline AMTGBot/Bot.cs]: Again, if following SOLID, one should pass this as an argument when creating an instance of Bot

File: AMTGBot/Bot.cs [hunk 3]
```diff
@@ -25,7 +32,10 @@ public async void SendSingleInlineQueryAnswer(string queryId, InlineQueryResultB
             {
                 await BotClient.AnswerInlineQueryAsync(queryId, new[] { baseResult });
             }
-            catch { }
+            catch (Exception e)
```
Comment (WhiteBlackGoose) [inline AMTGBot/Bot.cs]: Meh, we shouldn't catch any `Exception`. What if it is, I don't know, BAD ALLOC exception? Or `Universe collapsed panic exception`? Instead, there should be caught only those exceptions thrown by the method. If something very bad happened, it's better to crash the bot (with catching Exception, logging, and rethrowing), than to keep working

**Review by WhiteBlackGoose — COMMENTED** [review]
A few minor comments

File: AMTGBot/Bot.cs (hunk 3 above)
Comment (TheSeems) [reply AMTGBot/Bot.cs]: Okay, now catching bad request exception (when query is too old to answer).

File: AMTGBot/Bot.cs (hunk 2 above)
Comment (TheSeems) [reply AMTGBot/Bot.cs]: Agree.

File: AMTGBot/Bot.cs (hunk 1 above)
Comment (TheSeems) [reply AMTGBot/Bot.cs]: That's true. Gonna make it ILogger.

**Review by WhiteBlackGoose — APPROVED** [review]
Nice job!

**Pull request merged**